Samples/FsTests/BinaryUnitTests.fs

```diff
@@ -354,4 +354,27 @@ let lookaheadAndSeekRaw () =
 
     fs.Dispose()
 
-    File.Delete testFile 
\ No newline at end of file
+    File.Delete testFile
+
+
+[<Test>]
+let testMany () =
```

Comment:
Can we renmae this to be like "testManyZeroOrMoreMatches" or something a bit more descriptive? Then we'll merge!